corrector.rb

```diff
@@ -0,0 +1,8 @@
+class Corrector
+  def correct_name(name)
+    name = name.capitalize
+    name_length = name.length
+    name = name[0, 10] if name_length > 10
+    name
```

Comment:
[OPTIONAL]

- In the ruby it will return automatically the last variable which we used inside the function, so line no. 6 is not necessary to write, removing that will also give the same result. You can improve the code like this:

```ruby

def correct_name(name)
  name = name.capitalize
  name = name[0, 10] if name.length > 10
end
```

Reply:
Thank you so much for the feedback @Dipeshtwis, it is very much appreciated.

Reply:
@Dipeshtwis after trying this change, linter isn't accepting it

Reply:
Hi @mckent05,

What kind of error is linter giving?